index.html

```diff
@@ -1808,77 +1894,252 @@ <h2>Authentication</h2>
+is useful is when a <a>DID Subject</a> chooses to invoke their capability to
+start a vehicle through the combined usage of a <a>verification method</a> and
+the "canStartCar" capability. In this example, the vehicle would be the 
+<em>verifier</em> and would need to verify the <a>verification method</a> 
+exists in the <code>capabilityInvocation</code> property. 
+      </p>
+                  
+      <p>
+A <a>DID document</a> MAY include an <code>capabilityInvocation</code> 
+property. The <code>capabilityInvocation/code> property is an additional 
+example of a <a>verification relationship</a>.
       </p>
 
       <p>
-There are at least two suggested methods for implementing Authorization and
-Delegation, which may be layered:
+Example:
       </p>
+                  
+      <pre class="example nohighlight" title="Capability invocation property
+      containing two verification methods">
+{
+  "@context": "https://www.w3.org/ns/did/v1", "id":
+  "did:example:123456789abcdefghi",
+  <span class="comment">...</span>
+  "capabilityInvocation: [
+    <span class="comment">// this method can be used to perform key agreement
+    as did:...fghi</span>
+    "did:example:123456789abcdefghi#keys-1",
+    <span class="comment">// this method is *only* authorized for capability
+    invocation usage, it may not</span>
+    <span class="comment">// be used for any other verification relationship,
+    so its full description is</span>
+    <span class="comment">// embedded here rather than using only a
+    reference</span>
+    {
+    "id": "did:example:123456789abcdefghi#keys-2",
+    "type": "Ed25519VerificationKey2018",
+    "controller": "did:example:123456789abcdefghi",
+    "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+    }
+  ],
+}
+  <span class="comment">...</span>
 
-      <ul>
-        <li>
-A <a>verifiable data registry</a> could implement a coarse-grained
-<code>controller</code> pattern by enabling <a>DID documents</a> to express
-the <a>DID</a> of another <a>DID controller</a> that controls it, or
-additionally,
-        </li>
+      </pre>
+    </section>
 
-        <li>
-A <a>verifiable data registry</a> could implement a capabilities-based approach enabling
-further fine-grained control of authorization and delegation.
-        </li>
-      </ul>
+    <section>
+      <h3>capabilityDelegation</h3>
+      <p>
+The <code>capabilityDelegation<code> property is used to express a
+<a>verification relationship</a> which an entity can use to grant capabilities
+as the <a>DID Subject</a> or on behalf of the <a>DID subject</a> to other
+<em>capability invokers</em>. The <em>verifier</em> of a
+<code>capabilityDelegation<code> attempt can check proof of delegation to 
+grant invocation of a capability on behalf of the <a>DID subject</a> by 
+checking if the <a>verification method</a> used with the proof is contained 
+in the <code>capabilityDelegation<code> of the <a>DID Document</a>.
+      </p>
```

Comment:
Same comment as above, language is a bit complex.

Reply:
Made a small attempt to simplify. Wasn't sure I got it right though. As above, let's see if we might be able to simplify the example in a second round PR, so that I'm not blocking others much longer.